instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/cache/concurrentlinkedhashmap/LinkedDeque.java

```diff
@@ -31,9 +31,9 @@
 
 /**
  * Linked list implementation of the {@link Deque} interface where the link pointers are tightly
- * integrated with the element. Linked deques have no capacity restrictions; they grow as necessary
- * to support usage. They are not thread-safe; in the absence of external synchronization, they do
- * not support concurrent access by multiple threads. Null elements are prohibited.
+ * integrated with the element. Linked dequeues have no capacity restrictions; they grow as
```

Comment:
This one was actually on purpose "deques", as in Java `Deque` interface, the double ended queue

Reply:
fixed.